app/services/container_dashboard_service.rb

```diff
@@ -229,11 +229,8 @@ def daily_network_metrics
   end
 
   def daily_provider_metrics
-    if @daily_metrics.blank?
-      @daily_metrics = VimPerformanceDaily.find_entries(:tz => @controller.current_user.get_timezone)
-                                          .where(:resource => (@ems || ManageIQ::Providers::ContainerManager.all))
-                                          .where('timestamp > ?', 30.days.ago.utc).order("timestamp")
-    end
-    @daily_metrics
+    @daily_metrics ||= Metric::Helper.find_for_interval_name('daily', nil, nil, @controller.current_user.get_timezone)
```

Comment:
Using kwargs for all but the first parameter would avoid this `nil, nil` -- I think that's what @chessbyte was suggesting.